package.json

```diff
@@ -91,7 +91,7 @@
     "xo": "0.24.0"
   },
   "peerDependencies": {
-    "react": "15.x.x || 16.x.x || 17.x.x"
+    "react": "17.x.x || 18.x.x"
```

Comment:
why are we dropping 16.x.x?

Reply:
it's because of usage of hooks. Tested the implementation based on react 16 class components lifecycles that doesn't work well with this effect. After switching to `useLayoutEffect` the style registry handles it properly. So this would be a breaking change that dropping both react 15 and 16 😿

Reply:
So `useLayoutEffect` behaves differently in 17+?

Reply:
Oh it should include `>= 16.8`, updated.